tutorials/continuous-deployment-pipeline-google-container-engine-with-codeship/index.md

```diff
@@ -0,0 +1,275 @@
+
+Save the `.env` file once these items are finished.
+
+### Step 3: Run initial deployment
+
+You need to push an image and create your clusters in GKE initially before you
+can set up a fully automated pipeline in Codeship. In the `hello-express` source
+code is a `bin` folder with `bash` scripts perform these tasks using the
+Codeship Jet CLI.
+
+1.  Make sure the scripts in the `bin` folder are executable. In a terminal
+    window, navigate to the project folder and run the following command:
+
+        chmod +x ./bin/**
+
+1.  Update the `image` line in the `codeship-service.yml` file with your Google
+    Cloud Platform project ID.
+
+        app:
+          build:
+            dockerfile: Dockerfile
+            image: gcr.io/YOUR_PROJECT_IDE/hello-express #update this line using your Google Cloud Platform project ID
+        ...
+
+1.  Update the `image_name` line in the `codeship_steps.yml` file with your
+    Google Cloud Platform project ID.
+
+        - name: build-image
+          service: app
+          command: echo "Build completed"
+        - name: push-image-with-sha
+          service: app
+          type: push
+          image_name: "gcr.io/YOUR_PROJECT_ID/hello-express" #update this line using your Google Cloud Platform project ID
+          image_tag: "{{printf \"%.8s\" .CommitID}}"
+          registry: https://gcr.io
+          dockercfg_service: codeship_gcr_dockercfg
+        - name: tag-as-master
+          service: app
+          type: push
+          tag: master
+          image_name: "gcr.io/YOUR_PROJECT_ID/hello-express" #update this line using your Google Cloud Platform project ID
+          image_tag: "master"
+          registry: https://gcr.io
+          dockercfg_service: codeship_gcr_dockercfg
+        - name: gke-initial-deployment
+          service: codeship_gce_service
+          tag: master
+          command: bin/deploy --create
+        ...
+
+This pipeline will build the Docker image, and then push to Google Container
+Registry with a tag using the first 8 characters of the commit SHA for every
+commit to the repository. The third and fourth step will run only if the branch
+is tagged as master.
+
+You will run this pipeline locally using the [Codeship Jet CLI](https://documentation.codeship.com/pro/builds-and-configuration/cli/). There
+will be no commit SHA from git, but you will pass in a string using a flag.
+
+    jet steps --ci-commit-id 1234ABCD --tag master
+
+This command will run the steps to build your container cluster for the initial
+setup. It will take a few minutes to complete. Once completed, navigate to the
+[Discovery](https://console.cloud.google.com/kubernetes/discovery) page in the
+Google Cloud Platform Console to verify the status is `ok`. Click the endpoint
+to open the application in the browser. You should see `Hello Express!`.
+
+### Step 4: Create a Codeship project
+
+Now that you have set up the container cluster and deployed the application, you
```

Comment:
s/container/Container Engine/

Reply:
We use `container cluster` as a concept name in GKE docs/tutorials so I think it's ok.

Reply:
Should I change these, or leave them then?

Reply:
Since we use this term in the Container Engine docs, it's fine with me to leave them as-is.